pkg/ui/app/containers/nodeGraphs.tsx

```diff
@@ -151,6 +151,30 @@ class NodeGraphs extends React.Component<NodeGraphsProps, {}> {
               </Axis>
             </LineGraph>
 
+            <LineGraph title="Backend Statement Execution Latency"
+                       subtitle="(Max Per Percentile)"
+                       tooltip={`The latency of backend statements executed over 10 second periods.
+                                 Percentiles are calculated on each node.`}
+                       sources={sources}>
+              <Axis format={ (n: number) => d3.format(".1f")(NanoToMilli(n)) } label="Milliseconds">
```

Comment:
Don't use the "format" or "label" options anymore (I should have removed them); instead, use `<Axis units={AxisUnits.Duration}>`, that does a much better job.